test/rulesets/Base/array.jl

```diff
@@ -105,3 +105,15 @@ end
     test_rrule(fill, 44.0, 4; check_inferred=false)
     test_rrule(fill, 2.0, (3, 3, 3))
 end
+
+@testset "extrema" begin
+    @testset "$f" for f in [maximum, minimum]
+        test_rrule(f, rand(10))
```

Comment:
Simply writing `test_rrule(extrema, rand(10))` fails. Should this work on functions which return tuples?